contracts/contracts/MACI.sol

```diff
@@ -259,26 +234,8 @@ contract MACI is IMACI, DomainObjs, Params, Utilities, Ownable(msg.sender) {
   }
 
   /// @inheritdoc IMACI
-  function mergeStateAqSubRoots(uint256 _numSrQueueOps, uint256 _pollId) public onlyPoll(_pollId) {
-    stateAq.mergeSubRoots(_numSrQueueOps);
-
-    // if we have merged all subtrees then put a block
-    if (stateAq.subTreesMerged()) {
-      subtreesMerged = true;
-    }
-  }
-
-  /// @inheritdoc IMACI
-  function mergeStateAq(uint256 _pollId) public onlyPoll(_pollId) returns (uint256 root) {
-    // remove block
-    subtreesMerged = false;
-
-    root = stateAq.merge(stateTreeDepth);
-  }
-
-  /// @inheritdoc IMACI
-  function getStateAqRoot() public view returns (uint256 root) {
-    root = stateAq.getMainRoot(stateTreeDepth);
+  function getStateTreeRoot() public view returns (uint256 root) {
+    root = InternalLazyIMT._root(lazyIMTData);
```

Comment:
Why do these methods for IMT start with underscore?

Reply:
cause they are internal functions of a library

Reply:
Is there a reason why they are internal? Maybe it makes sense to change them to public like for poseidon libs?

Reply:
https://github.com/privacy-scaling-explorations/zk-kit/blob/main/packages/imt.sol/contracts/internal/InternalLazyIMT.sol

Reply:
right so I'm trying to convert them to public, the next step is to use the directive `use InternalLazyIMT for lazyIMTData` -> though for this I would need to create a wrapper contract which calls these functions, deploy with linked library, then inherit from this contract within MACI, when we can directly call the library 🤔  Reckon it's something worth doing?

Reply:
Let's keep it simple instead :)